shared/firestore-models/repo-config.ts

```diff
@@ -43,3 +43,13 @@ export async function getBranchByLabel(
     return config.branches.find(branch => branch.label === label);
   }
 }
+
+/** Determine if a label is one of the target labels based on the repos config. */
+export async function getBranchByBranchName(
```

Comment:
😄 how about just `getBranchByName`? even though the property is `branchName`

Reply:
added TODO to decide later.